chipsec/hal/spi_uefi.py

```diff
@@ -263,7 +263,10 @@ def build_efi_modules_tree( _uefi, fwtype, data, Size, offset, polarity ):
                 sec.calc_hashes( sec.HeaderSize )
             elif sec.Type == EFI_SECTION_USER_INTERFACE:
                 # "leaf" UI section: update section's UI name
-                sec.ui_string = sec.Image[sec.HeaderSize:-2].decode("utf-16")
+                try:
+                    sec.ui_string = sec.Image[sec.HeaderSize:-2].decode("utf-16")
+                except UnicodeDecodeError:
+                    pass
```

Comment:
Should you set the ui_string to something like "unknown" if it can't read it from the image?

Reply:
I don't know that a blank ui_string vs an unknown ui_string would be helpful.  Can make changes if others have consensus that it needs to be changed